jbehave-support-core/src/main/java/org/jbehavesupport/core/report/extension/ServerLogXmlReporterExtension.java

```diff
@@ -1,33 +1,76 @@
 package org.jbehavesupport.core.report.extension;
 
+import java.io.File;
+import java.io.IOException;
 import java.io.Writer;
-import java.util.ArrayList;
-import java.util.Arrays;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Date;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
-import java.util.stream.Collectors;
 
 import lombok.RequiredArgsConstructor;
-import org.jbehavesupport.core.report.ReportContext;
-import org.jbehavesupport.core.ssh.SshLog;
-import org.jbehavesupport.core.ssh.SshTemplate;
-
+import org.apache.commons.collections4.keyvalue.MultiKey;
+import org.apache.commons.collections4.map.LRUMap;
+import org.apache.commons.collections4.map.MultiKeyMap;
 import org.apache.commons.lang3.exception.ExceptionUtils;
-import org.springframework.beans.factory.annotation.Qualifier;
-import org.springframework.beans.factory.config.BeanDefinition;
-import org.springframework.beans.factory.config.ConfigurableListableBeanFactory;
-import org.springframework.beans.factory.support.RootBeanDefinition;
+import org.codehaus.plexus.util.FileUtils;
```

Comment:
Just a question - can we use spring or Apache file utils instead?

Reply:
We can :)